README.md

```diff
@@ -1,2 +1,17 @@
-# module-1-first-repository
- 
+# New Project as example!
+
+Hello!
+
+> Today I'm creating my first repository of Module 1 from Microverse.
+
+On this proyect I will work on the next concepts:
+
+- Git & Github Overflow
+- Linters for HTML, CSS and JS
+- Create Branch whit best practices
+- Best practices of commiting
+- Best practices for Pull Request
+
+Live Demo (Work on progress).
+
+![Esta es una imagen](https://camo.githubusercontent.com/38f113b96a368dfb7f634d2f2da97e7b8c748042d2a284b97c3fad048bb3ff55/68747470733a2f2f6d69726f2e6d656469756d2e636f6d2f6d61782f323733322f312a6d74736b3366515f4252656d466964686b656c3364412e706e67)
```

Comment:
- [ ] Good job adding a README.md file to your project, however, it's not descriptive enough. Please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.

Reply:
Hi phelian23, thank you for reviewing my code! Ok, I'm trying to get working linters, but I'm stuck. This is the error that I'm getting.
![image](https://user-images.githubusercontent.com/93630700/164105856-0fb46d6c-b809-40e1-9d4a-d0a30647492c.png)

 I'm using Prettier, I'ts not useful as linters?. I have updated the readme whit more info.

Reply:
@phelian23, I was not adding the correct files, that's why the linters were not working for me.